<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMSReplay</class>
 <widget class="QMainWindow" name="FMSReplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FMSReplay</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1300</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Top">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="pBtn_Open">
        <property name="icon">
         <iconset theme="fileopen">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_FileName">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;lt;FMS log file not loaded&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_PStart">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>PROGRAM START:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnEd_SetProgStartTime">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="inputMask">
         <string>99.99.9999 / 99:99:99.999</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Ignition">
        <property name="text">
         <string>IGNITION:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_IgnitionTime">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_ZoomReset">
        <property name="text">
         <string>Zoom reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pBtn_Help">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_Main">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_Tabs">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>0</height>
           </size>
          </property>
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_Log">
           <attribute name="title">
            <string>FMS log</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="leftMargin">
             <number>9</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="pTxtEd">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Analiz">
           <attribute name="title">
            <string>Report</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QTableWidget" name="tblWidget">
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Время</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Параметр</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Значение</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Регистр</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>New Column</string>
                </property>
               </column>
              </widget>
              <widget class="QPlainTextEdit" name="pTxtEd_Report">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_LevelF1">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Level F1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QChartView" name="graphView_F">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_LevelO1">
           <attribute name="title">
            <string>Level O1</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QChartView" name="graphView_O"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_LevelF1_UTC">
           <attribute name="title">
            <string>Level F1(UTC)</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QChartView" name="graphView_FU"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_LevelO1_UTC">
           <attribute name="title">
            <string>Level O1 (UTC)</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QChartView" name="graphView_OU"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_TempF1_UTC">
           <attribute name="title">
            <string>Temperature (UTC)</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QChartView" name="graphView_TFU"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
